### DISCUSSION TITLE
Asynchronous Shaping - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#799] [Showcases]
Discuss the "Asynchronous Shaping" showcase here. Ideas and recommendations are welcome.

**Comment by madhusrivatsav** [comment]
Can you please share some references on how to make a custom eligibility time meter?

**Comment by NunzioFornitto** [comment]
Tag 'inet::EligibilityTimeTag' is absent -- in module (inet::EligibilityTimeGate) TestNet.switch0.eth[0].macLayer.queue.transmissionSelectionAlgorithm[3]
I have this error at the beginning of the simulation. I followed the guide on Inet documentation
